config/thoth.yaml

```diff
@@ -105,6 +105,23 @@ repositories:
           labels: [bot]
           changelog_file: true
 
+  - slug: thoth-station/graph-sync-scheduler
+    token: '{KEBECHET_TOKEN}'
+    managers:
+      - name: update
+        configuration:
+          labels: [bot]
+      - name: info
+      - name: version
```

Comment:
No need to have version manager (we do not release it on PyPI), update (and info) manager are good enough. I will update the PR.